mediagrains/tools/wrap_in_gsf.py

```diff
@@ -164,11 +169,10 @@ def wrap_audio_in_gsf():
         if sample_rate is None:
             sample_rate = 48000
 
-        grain_rate = fractions.Fraction(sample_rate, samples_per_grain)
-
         template_grain = AudioGrain(
             flow_id=flow_id, source_id=source_id, origin_timestamp=args.start_ts, sample_rate=sample_rate,
-            rate=grain_rate, channels=channels, samples=samples_per_grain, cog_audio_format=args.format
+            duration=duration, channels=channels, samples=samples_per_grain, cog_audio_format=args.format,
```

Comment:
The duration should be the grain_rate line changed to
`duration = fractions.Fraction(samples_per_grain, sample_rate,)`